### PR TITLE
added AraTha results

### PR DESCRIPTION
**Pull request by andrewkern**

### CODE REVIEW

**Comment by andrewkern** [comment]
hoping @petrelharp @ckyriazis @agladstein @dschride could look at this quickly

File: popSimManu.tex [hunk 1]
```diff
@@ -473,11 +473,15 @@ \subsection*{Use case: comparing methods of demographic inference}
...
+model, originally estimated from an African sample of \textit{D.~melanogaster} \citep{sheehan2016deep}.
+As with humans, we use \stdpopsim to simulate replicate \textit{Drosophila} genomes under
 an empirically derived genetic map, and then try to infer back the parameters of the model.
 Accuracy is mixed among methods in this setting and generally worse than what we
-observe under simulations of the human genome.
+observe under simulations of the human genome. In Figure \ref{fig:n_t_AraTha} we
+show the equivalent information but here from simulations of \textit{A.~thaliana}
+under the African2Epoch\_1H18 model originally inferred by \cite{huber2018gene}.
+As in \textit{D.~melanogaster} simulations, the performance of these methods 
+in \textit{A.~thaliana} suffers in comparison to what is observed in humans.
```
Comment (petrelharp) [inline popSimManu.tex]: Wordy; how about
```
As most method development for population genetics has been focused on human
data, it is of consequence to ask how such methods might perform in non-human
genomes. Figure \ref{fig:n_t_sheehan} shows parameter estimates from the African3Epoch\_1S16
model, originally estimated from an African sample of \textit{D.~melanogaster} \citep{sheehan2016deep},
and Figure \ref{fig:n_t_AraTha} shows estimates from simulations of \textit{A.~thaliana}
under the African2Epoch\_1H18 model originally inferred by \cite{huber2018gene}.
In both cases, as with humans, we use \stdpopsim to simulate replicate genomes using an empirically derived genetic map,
and try to infer back parameters of the simulation model.
Accuracy is mixed among methods in this setting and generally worse than what we
observe for simulations of the human genome.
```

**Comment by petrelharp** [comment]
This would close #130.

File: popSimManu.tex (hunk 1 above)
Comment (andrewkern) [reply popSimManu.tex]: this is great. will add

File: popSimManu.tex [hunk 2]
```diff
@@ -726,11 +730,11 @@ \subsection*{Workflow for analysis of simulated data}
...
 we estimated $N(t)$ from an African sample simulated under the DroMel,
 African3Epoch\_1S16 model using the Comeron2012\_dm6 map.
-% RNG: No results from these simulations are included in the paper
-%Finally we ran simulations of \textit{A.~thaliana} genomes using the AraTha African2Epoch\_1H18 model under the Salome2012\_TAIR7 map.
-For each model, we estimated population size histories
-separately from each of three replicate simulations of whole genome datasets.
-In all cases we simulated 50 sampled chromosomes in the focal population.
+Finally we ran simulations of \emph{A. thaliana} genomes using the 
+AraTha African2Epoch\_1H18 model under the Salome2012\_TAIR7 map.
+For each model, three replicate whole genomes were
```
Comment (jradrion) [inline popSimManu.tex:735]: might want to add the "demog_tuple" parameters for AraTha African2Epoch_1H18 (i.e. [500,1000])

**Review by jradrion — APPROVED** [review]
...everything else lgtm

File: popSimManu.tex (hunk 2 above)
Comment (andrewkern) [reply popSimManu.tex:735]: it's all specified in the catalog

**Comment by andrewkern** [comment]
okay gonna merge these edits

**Pull request merged**